clang/lib/Interpreter/RemoteJITUtils.cpp

```diff
@@ -0,0 +1,263 @@
+//===-- RemoteJITUtils.cpp - Utilities for remote-JITing --------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
This is copy/pasted from the llvm-jitlink copy, right?

I wonder if we can come up with a central place for this so that we can re-use. It's probably not too bad to copy/paste for now though.

Reply:
Could we move this to the llvm Support?

Reply:
Maybe to ORC?

Reply:
Let's leave it for now, but add a FIXME to unify the code with the stuff in `llvm-jitlink`. ORC is the right place for it.